{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/conftest.py

```diff
@@ -3,47 +3,37 @@
 import os
 
 from astropy.version import version as astropy_version
+
+# For Astropy 3.0 and later, we can use the standalone pytest plugin
 if astropy_version < '3.0':
-    # With older versions of Astropy, we actually need to import the pytest
-    # plugins themselves in order to make them discoverable by pytest.
-    from astropy.tests.pytest_plugins import *
+    from astropy.tests.pytest_plugins import *  # noqa
+    del pytest_report_header
 else:
-    # As of Astropy 3.0, the pytest plugins provided by Astropy are
-    # automatically made available when Astropy is installed. This means it's
-    # not necessary to import them here, but we still need to import global
-    # variables that are used for configuration.
-    from astropy.tests.plugins.display import (pytest_report_header,
-                                               PYTEST_HEADER_MODULES,
-                                               TESTED_VERSIONS)
-
-from astropy.tests.helper import enable_deprecations_as_exceptions
-
-## Uncomment the following line to treat all DeprecationWarnings as
-## exceptions. For Astropy v2.0 or later, there are 2 additional keywords,
-## as follow (although default should work for most cases).
-## To ignore some packages that produce deprecation warnings on import
-## (in addition to 'compiler', 'scipy', 'pygments', 'ipykernel', and
-## 'setuptools'), add:
-##     modules_to_ignore_on_import=['module_1', 'module_2']
-## To ignore some specific deprecation warning messages for Python version
-## MAJOR.MINOR or later, add:
-##     warnings_to_ignore_by_pyver={(MAJOR, MINOR): ['Message to ignore']}
-# enable_deprecations_as_exceptions()
+    from pytest_astropy_header.display import PYTEST_HEADER_MODULES, TESTED_VERSIONS
```

Comment:
This probably should also be in a `try/except` - there is no need to insist that packages have this installed. Maybe move it inside the `pytest_configure()`?

Reply:
I can't easily move it inside the pytest_configure because the first clause of the 'if' does need to be outside of the function, as it imports ``pytest_*`` functions. However I've adjusted the code so that it should work even if the plugin isn't installed.